src/dagmc/dagmcmetadata.cpp

```diff
@@ -337,10 +345,17 @@ void dagmcMetaData::parse_boundary_data() {
     // 2d entities have been tagged with the boundary condition property
     // ie. both surfaces and its members triangles,
 
-    if (boundary_assignment[0].find("Reflecting") != std::string::npos)
+
+    std::string bc_string = to_lower(boundary_assignment[0]);
+
+    if (bc_string.find("reflecting") != std::string::npos)
```

Comment:
Does it make sense to unmagic these strings like we've done with `graveyard_str` and `vacuum_str`?

Reply:
Yeah I'd say so. Can do!